annotations/views.py

```diff
@@ -183,6 +190,25 @@ def user_settings(request):
     })
     return HttpResponse(template.render(context))
 
+
+def save_file(f):
```

Comment:
Hmm, did you look at the [File Uploads](https://docs.djangoproject.com/en/1.9/topics/http/file-uploads/#handling-uploaded-files-with-a-model) documentation? The file should be saved when the form is saved, so you may not need this function at all.